salt/metalk8s/salt/master/files/salt-master-manifest.yaml.j2

```diff
@@ -58,12 +58,14 @@ spec:
           mountPath: '/var/cache/salt'
         - name: run
           mountPath: '/var/run/salt'
-        - name: states
-          mountPath: '{{ iso_root_path }}/salt'
+        {%- for version, path in iso_root_path.items() | sort(attribute='0') %}
```

Comment:
Ok, that doesn't work for the moment, need to inverstigate this one 

As we only got the `iso_root_path` of the last pillarenv in the pillar.